web/client/components/mapcontrols/scale/ScaleBox.jsx

```diff
@@ -49,9 +48,9 @@ var ScaleBox = React.createClass({
     render() {
         let control = this.props.readOnly ?
             <label>{this.props.template(this.props.scales[this.props.currentZoomLvl], this.props.currentZoomLvl)}</label>
-        : <Input type="select" label={this.props.label} onChange={this.onComboChange} bsSize="small" value={this.props.currentZoomLvl}>
```

Comment:
Also here: Input works for us, before replacing it we need to understand when it doesn't work